sdk/webpubsub/Microsoft.Azure.WebPubSub.AspNetCore/README.md

```diff
@@ -38,25 +38,11 @@ In order to interact with the service, you'll need to inject the Web PubSub serv
 ### Inject Web PubSub service with options
 
 ```C# Snippet:WebPubSubDependencyInjection
-public void ConfigureServices(IServiceCollection services)
```

Comment:
This deletes snippets from teh README.  I think it is better to revert and guide the webpubsub through the process of updating snippets correctly, per https://github.com/Azure/azure-sdk-for-net/pull/27360

Reply:
The snippets were reporting back as unused when I ran codegen.

```text
System.InvalidOperationException: Not all snippets were used.
webpubsub: Snippet:WebPubSubAddUserToGroup
webpubsub: Snippet:Managing_WebPubSub_GetWebPubSubIfExists
webpubsub: Snippet:WebPubSubDefaultMethods
   at Azure.Sdk.Tools.SnippetGenerator.Program.OnExecuteAsync() in /mnt/vss/_work/1/s/tools/snippet-generator/Azure.Sdk.Tools.SnippetGenerator/Program.cs:line 51
   at McMaster.Extensions.CommandLineUtils.Conventions.ExecuteMethodConvention.InvokeAsync(MethodInfo method, Object instance, Object[] arguments)
   at McMaster.Extensions.CommandLineUtils.Conventions.ExecuteMethodConvention.OnExecute(ConventionContext context)
   at McMaster.Extensions.CommandLineUtils.Conventions.ExecuteMethodConvention.<>c__DisplayClass0_0.<<Apply>b__0>d.MoveNext()
--- End of stack trace from previous location ---
   at McMaster.Extensions.CommandLineUtils.CommandLineApplication.<>c__DisplayClass142_0.<OnExecute>b__0()
   at McMaster.Extensions.CommandLineUtils.CommandLineApplication.Execute(String[] args)
   at McMaster.Extensions.CommandLineUtils.CommandLineApplication.Execute[TApp](CommandLineContext context)
   at McMaster.Extensions.CommandLineUtils.CommandLineApplication.Execute[TApp](IConsole console, String[] args)
   at McMaster.Extensions.CommandLineUtils.CommandLineApplication.Execute[TApp](String[] args)
   at Azure.Sdk.Tools.SnippetGenerator.Program.Main(String[] args) in /mnt/vss/_work/1/s/tools/snippet-generator/Azure.Sdk.Tools.SnippetGenerator/Program.cs:line 68
```

Reply:
I agree.   This looks wrong.   Let me troubleshoot.

Reply:
@JoshLove-msft  figured it out: https://github.com/Azure/azure-sdk-for-net/pull/27360#issuecomment-1059455596